openedx/core/types.py

```diff
@@ -0,0 +1,3 @@
+import django.contrib.auth.models
```

Comment:
Cool! Works for me.

Name suggestion: `openedx/core/typing.py`, to parallel the built-in `typing.py` module. It would also imply a bit more strongly that this module is specifically for typing utilities, and not just any old class.

Reply:
On the other hand... this wouldn't *just* be for type checking, as we'd expect `from openedx.core.types import User` to be the standard way of referencing the User model.

Eh, whichever you prefer is fine by me.